infrastructure/staking-mechanics/offenses-and-slashes.md

```diff
@@ -1,4 +1,179 @@
+
+#### Equivocation Slash
+
+The Web3 Foundation's [Slashing mechanisms](https://research.web3.foundation/Polkadot/security/slashing/amounts){target=\_blank} page provides guidelines for evaluating the security threat level of different offenses and determining penalties proportionate to the threat level of the offense. Offenses requiring coordination between validators or extensive computational costs to the system will typically call for harsher penalties than those more likely to be unintentional than malicious. A description of potential offenses for each threat level, and the corresponding penalties, is as follows:
+
+- **Level 1** - honest misconduct such as isolated cases of unresponsivness
+    - **Penalty** - validator can be kicked out or slashed up to 0.1% of stake in the validator slot
+- **Level 2** - misconduct that can occur honestly but is a sign of bad practices. Examples include repeated cases of unresponsiveness and isolated cases of equivocation
+    - **Penalty** - slash of up to 1% of stake in the validator slot
+- **Level 3** - misconduct that is likely intentional but of limited affect on the performance or security of the network. This level will typically include signs of coordination between validators. Examples include repeated cases of equivocation or isolated cases of unjustified voting on GRANDPA
+    - **Penalty** - reduction in networking reputation metrics, slash of up to 10% of stake in the validator slot
+- **Level 4** - misconduct that poses severe security or monetary risk to the system or mass collusion. Examples include signs of extensive coordination, creating a serious security risk to the system, or forcing the system to use extensive resources to counter the misconduct
+    - **Penalty** - slash of up to 100% of stake in the validator slot
+
+See the next section to understand how slash amounts for equivocations are calculated. If you want to know more details about slashing, please look at the research page on [Slashing mechanisms](https://research.web3.foundation/Polkadot/security/slashing/amounts){target=\_blank}.
+
+#### Slash Calculation for Equivocation
+
+The slashing penalty for GRANDPA, BABE, and BEEFY equivocations is calculated using the formula below, where `x` represents the number of offenders and `n` is the total number of validators in the active set:
+
+```text
+min((3 * x / n )^2, 1)
+```
+
+The following scenarios demonstrate how this formula means slash percentages can increase exponentially based on the number of offenders involved compared to the size of the validator pool:
+
+- **Minor offense** - assume one validator out of a 100 validator active set equivocates in a slot. A single validator committing an isolated offense is most likely a mistake rather than malicious attack on the network. This offense results in a 0.09% slash to the stake in the validator slot
```

Comment:
```suggestion
- **Minor offense** - assume 1 validator out of a 100 validator active set equivocates in a slot. A single validator committing an isolated offense is most likely a mistake rather than malicious attack on the network. This offense results in a 0.09% slash to the stake in the validator slot
```